tests/js/PrettierLintBearTest.py

```diff
@@ -14,6 +14,40 @@
 """
 # Stop ignoring
 
+bad_semicolon = """
+var x;
+"""
+
+good_quotes = """const a = () => "Foo bar";
+
+function b(object, key) {
+  return object["key"];
+}"""
+
+bad_quotes = """const a = () => 'Foo bar';"""
+
```

Comment:
Remove the extra newline present in line 28 and 35. To make your code pretty :smile: